chandra_aca/aca_image.py

```diff
@@ -828,3 +832,82 @@ def get_psf_image(
 
         out = ACAImage(psf, row0=row0, col0=col0) if aca_image else (psf, row0, col0)
         return out
+
+
+@retry.retry(exceptions=requests.exceptions.RequestException, delay=5, tries=3)
+def get_aca_image_table(
+    start: CxoTimeLike, stop: CxoTimeLike, bgsub=True, source="maude", **maude_kwargs
+):
+    """
+    Get ACA images and ancillary data from either the mica or maude data sources.
+
+    Parameters
+    ----------
+    start : CxoTimeLike
+        start time
+    stop : CxoTimeLike
+        stop time (CXC sec)
+    bgsub : bool
+        include background subtracted images in output table
+    source : str
+        'maude' or 'mica'
+    maude_kwargs
+        additional kwargs for maude data source
+
+    Returns
+    -------
+    Table of ACA images and ancillary data. If bgsub is True then the table
```

Comment:
Follow normal NumpyDoc standard like in Parameters. For long-form information like the new columns, put that in the description above.